app/modules/transcribe/subjects.factory.js

```diff
@@ -93,6 +94,14 @@ function SubjectsFactory($q, AnnotationsFactory, localStorageService, zooAPI, zo
         }
     }
 
+    function _getSubjectSetId() {
+        if (_subjectSet) {
+          return Promise.resolve(_subjectSet);
+        } else {
+          return _getRandomWorkflowAssociatedSubjectSets()
```

Comment:
So looks like this won't sticky the random set selection if i click get started. Should this be the default behaviour or is it ok to flip between genre's if i don't select one?

Reply:
I don't see why that would be a problem and it should have been the case already before this fix. But it might be worth asking in channel.